llvm/test/CodeGen/RISCV/cheri/global-cap-import-attributes.ll

```diff
@@ -0,0 +1,323 @@
+; RUN: llc --filetype=asm --mcpu=cheriot --mtriple=riscv32-unknown-unknown -target-abi cheriot -mattr=+xcheri,+cap-mode < %s | FileCheck %s
```

Comment:
Triple should be `riscv32cheriot-unknown-cheriotrtos`

Reply:
[2a7ee40](https://github.com/CHERIoT-Platform/llvm-project/pull/147/commits/2a7ee406925896e121e8e054deb61045cdf343ef) changes the triple.